sdk/framework/openapi_test.go

```diff
@@ -18,75 +18,6 @@ import (
 )
 
 func TestOpenAPI_Regex(t *testing.T) {
-	t.Run("Required", func(t *testing.T) {
```

Comment:
Are all of these tests just not doing anything any more, or is there a need for more test cases to be added back in place of these?

Reply:
All of the tests being deleted, test the behaviour of various ad-hoc regexps, which were being used to parse other regexps (!).

They have all been deleted, and their job replaced by `regexp/syntax` from the Go standard library.

The new code that has been written to process the output of the stdlib's `regexp/syntax` is already well-exercised by the existing `TestOpenAPI_ExpandPattern` - although, I could add some additional inputs and expected outputs to that test, which would fail with the old implementation, but pass with this rewritten one.

Reply:
>  I could add some additional inputs and expected outputs to that test, which would fail with the old implementation, but pass with this rewritten one.

These would be awesome! I would also love to see some of the edge cases tested like ".*" that you mentioned in the comments.

Reply:
Done!